internal/provider/ovirt/ovirt_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package ovirt
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestOvirt(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "auth tests")
```

Comment:
What does it mean?

Reply:
specifically if this really related to authentication

Reply:
typo, copy from another package, will change